### PR TITLE
Appended Quote

### PR DESCRIPTION
**Pull request by Lalisfeed**
## ✅ New Quote Checklist
- [X] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [X] The quote(s) added are about **learning a foreign language**.
- [X] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [X] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
The quote's author should be a person. Please change it and make a PR by properly formatting and following the [guidelines](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

File: server/data/quotes.json [hunk 1]
```diff
@@ -304,7 +304,7 @@
     "sentence": "Who wants to have an effect on others, must first speak in their language with them."
   },
   {
-	  "author": "A Quote For You",
+	  "author": "Alexis fledge",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:307]: Please capitalize the letter "F" in the word Fledge.

Comment (Lalisfeed) [reply server/data/quotes.json]: Done

File: server/data/quotes.json [hunk 2]
```diff
@@ -303,6 +303,10 @@
     "author": "Kurt Tucholsky",
     "sentence": "Who wants to have an effect on others, must first speak in their language with them."
   },
+  {
+    "author": "Alexis Fledge",
+    "sentence": "A man who experienced more than one language has experienced more than one Culture."
```
Comment (jayehernandez) [inline server/data/quotes.json:308]: Remove capitalization for Culture
```suggestion
    "sentence": "A man who experienced more than one language has experienced more than one culture."
```

**Review by jayehernandez — CHANGES_REQUESTED** [review]

File: server/data/quotes.json (hunk 2 above)
Comment (Lalisfeed) [reply server/data/quotes.json]: did remove and inserted the quote into the dictionary order

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @Lalisfeed!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**